clients/js-solita-candy-guard/src/generated/types/AllocationTracker.ts

```diff
@@ -6,15 +6,15 @@
  */
 
 import * as beet from '@metaplex-foundation/beet';
-export type MintTracker = {
+export type AllocationTracker = {
```

Comment:
On a previous commit, I saw you made this type an account on Kinobi so it could generate PDA helpers. Any reason you reverted this?

Reply:
It is there, no?
![image](https://github.com/metaplex-foundation/mpl-candy-machine/assets/729235/f238978f-a18b-4fa2-a92a-8879a0836888)

Reply:
As discussed, this is actually in Solita so we're good.